Libraries/Text/RCTTextKeyValueConstants.h

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+extern NSString *const kNewlineRawValue;
```

Comment:
I made them constants mainly because we have to use them in both RCTTextField & RCTTextView, I didn't want that duplication. Also I added them because I thought they were more explicit that their string values. I'm not married to this though and can remove them if preferred.